docs/src/main/asciidoc/reactive-sql-clients.adoc

```diff
@@ -523,6 +523,66 @@ return SqlClientHelper.inTransactionUni(client, tx -> tx
         .onItem().ignore().andContinueWithNull());
 ----
 
+== UNIX Domain Socket connections
+
+The PostgreSQL and MariaDB/MySQL clients can be configured to connect to the server through a UNIX domain socket.
+
+First make sure that https://quarkus.io/guides/vertx#native-transport[native transport support] is enabled.
+
+Then configure the database connection url.
+This step depends on the database type.
+
+=== PostgreSQL
+
+PostgresSQL domain socket path have the following form: `<directory>/.s.PGSQL.<port>`
+
+The database connection url must be configured so that:
+
+* the `host` is the `directory` in the socket path
+* the `port` is the `port` in the socket path
+
+Consider the following socket path: `/var/run/postgresql/.s.PGSQL.5432`.
+
+In `application.properties` add:
+
+[source,properties]
+----
+quarkus.datasource.reactive.url=postgresql://:5432/quarkus_test?host=/var/run/postgresql
+----
+
+Or in `application.yml`:
+
+[source,yaml]
+----
+quarkus:
+  datasource:
+    reactive:
+      url: postgresql://:5432/quarkus_test?host=/var/run/postgresql
+----
```

Comment:
Can you drop the YAML part? We don't have it in the other guides and we definitely don't want to have that everywhere (except if one day, we have a nice tab feature allowing to display both nicely).

Reply:
Done